zsh-aws-vault.plugin.zsh

```diff
@@ -6,6 +6,8 @@ AWS_VAULT_PL_CHAR=${AWS_VAULT_PL_CHAR:-$'\u2601'} # "the cloud"
 AWS_VAULT_PL_BROWSER=${AWS_VAULT_PL_BROWSER:-''}
 AWS_VAULT_PL_BROWSER_LAUNCH_OPTS=${AWS_VAULT_PL_BROWSER_LAUNCH_OPTS:-''}
 AWS_VAULT_PL_MFA=${AWS_VAULT_PL_MFA:-''}
+AWS_VAULT_PL_PERSIST_PROFILE=${AWS_VAULT_PL_PERSIST_PROFILE:-false}
```

Comment:
#40 was using a slightly different approach here. I prefer setting booleans and having a separate variable for the boolean switch and the path. I find this pattern slightly more explicit.